sc-platform-api/src/security/roles/admin.guard.ts

```diff
@@ -0,0 +1,15 @@
+import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
+import { Observable } from 'rxjs';
+import { Role } from '@prisma/client';
+
+@Injectable()
+export class AdminGuard implements CanActivate {
+  canActivate (
+    context: ExecutionContext
+  ): boolean | Promise<boolean> | Observable<boolean> {
+    const request = context.switchToHttp().getRequest();
```

Comment:
Request has type of `any` which is bad practise, so add type of `Request` for it
```suggestion
    const request = context.switchToHttp().getRequest<Request>();
```

Reply:
Should i create custom Request type? Because there is no user in default Request type